src/components/layout/CyVerseAppBar.js

```diff
@@ -230,6 +232,10 @@ function CyverseAppBar(props) {
     const [bootstrapQueryEnabled, setBootstrapQueryEnabled] = useState(false);
     const [profileRefetchInterval, setProfileRefetchInterval] = useState(null);
     const setPreferences = usePreferences()[1];
+    mockAxios
```

Comment:
You added mockAxios directly to the CyverseAppBar component - this is incorrect.  mockAxios stuff is for `storybook` only.  mockAxios fakes api calls so `storybook` can simulate real api calls. Adding this fake stuff to a real component (*.js file instead of *.story file) breaks the real component outside of `storybook`.

Reply:
Yup. Am trying to run this in local machine and everything is failing.